simpleclient/src/main/java/io/prometheus/client/Histogram.java

```diff
@@ -207,6 +208,24 @@ public double time(Runnable timeable) {
       return elapsed;
     }
 
+    /**
+     * Executes callable code (i.e. a Java 8 Lambda) and observes a duration of how long it took to run.
+     *
+     * @param timeable Code that is being timed
+     * @return Result returned by callable.
+     */
+    public <E> E time(Callable<E> timeable) {
+      Timer timer = startTimer();
+
+      try {
+        return timeable.call();
+      } catch (Exception e) {
+        throw new RuntimeException(e);
```

Comment:
Why are you wrapping this exception rather than just letting it propagate naturally?